src/gui/reports/ReportsWidgetHealthcheck.cpp

```diff
@@ -180,11 +204,21 @@ void ReportsWidgetHealthcheck::addHealthRow(QSharedPointer<PasswordHealth> healt
     }
 
     auto row = QList<QStandardItem*>();
-    row << new QStandardItem(descr);
-    row << new QStandardItem(entry->iconPixmap(), title);
-    row << new QStandardItem(group->iconPixmap(), group->hierarchy().join("/"));
-    row << new QStandardItem(QString::number(health->score()));
-    row << new QStandardItem(health->scoreReason());
+    descrItem = new QStandardItem(descr);
+    descrItem->setData(health->score(), Qt::UserRole);
+    row << descrItem;
+    titleItem = new QStandardItem(entry->iconPixmap(), title);
+    titleItem->setData(title, Qt::UserRole);
+    row << titleItem;
+    pathItem = new QStandardItem(group->iconPixmap(), group->hierarchy().join("/"));
+    pathItem->setData(group->hierarchy().join("/"), Qt::UserRole);
+    row << pathItem;
+    scoreItem = new QStandardItem(QString::number(health->score()));
+    scoreItem->setData(health->score(), Qt::UserRole);
+    row << scoreItem;
+    reasonItem = new QStandardItem(health->scoreReason());
+    reasonItem->setData(health->scoreReason(), Qt::UserRole);
+    row << reasonItem;
```

Comment:
With ReportSortProxyModel we now sort rows based on their data and not on their Qt::UserRole.
Do we still need each column to have a setdata with Qt::UserRole?
```suggestion
    row << new QStandardItem(descr)
        << new QStandardItem(entry->iconPixmap(), title)
        << new QStandardItem(group->iconPixmap(), group->hierarchy().join("/"))
        << new QStandardItem(QString::number(health->score()))
        << new QStandardItem(health->scoreReason());
```